src/Arriba/Arriba/Diagnostics/Tracing/ArribaLogs.cs

```diff
@@ -16,5 +16,9 @@ public void WriteLine(string message, params object[] args)
             Console.WriteLine(message, args);
         }
 
+        public void WriteLine(string message)
```

Comment:
Why add this?

Reply:
After getting the changes merged I was getting an error  just during execution at this call on OAuthController.ReadTokenResultAsync

``` 
_log.WriteLine($"ReadTokenResultAsync result: {result}");
```

Instead of changing this call I decided to add a method that receives the string

Reply:
hmm.. weird to see it be required, but not to see a change that uses is..

Reply:
I did change the usage of the logger and removed the method overload